guides/upgrading/index.md

```diff
@@ -1,39 +1,27 @@
-When someone says they are "upgrading" their Ember app, it could mean one of several things, especially if they say that they are "upgrading to Octane."
-This Guide will help fill in the blanks about how to upgrade your app's version, get access to the latest and greatest features, and form a strategy for using Octane's features in your existing apps.
+Quand des devs disent qu'ils "mettent à jour" leur app Ember, ça peut vouloir dire plusieurs choses, en particulier s'ils parlent de "passer à Octane". Cette page du Guide vous aidera à remplir les blancs sur ce qui concerne la mise à jour la version de votre app, l'accès aux dernières et meilleures fonctionnalités, et à définir une stratégie pour utiliser les fonctionnalités d'Octane dans vos apps existantes.
```

Comment:
```suggestion
Quand certains disent qu'ils "mettent à jour" leur app Ember, ça peut vouloir dire plusieurs choses, en particulier s'ils parlent de "passer à Octane". Cette page du Guide vous aidera à remplir les blancs sur ce qui concerne la mise à jour la version de votre app, l'accès aux dernières et meilleures fonctionnalités, et à définir une stratégie pour utiliser les fonctionnalités d'Octane dans vos apps existantes.
```

Nitpicking: I would keep the same line breaks at it makes the reading easier, less compact.